src/components/find-on-shelf/FindOnShelfModalBody.tsx

```diff
@@ -54,7 +54,7 @@ const FindOnShelfModalBody: FC<FindOnShelfModalBodyProps> = ({
   );
   const { data, isLoading } = useGetHoldings({
     faustIds: faustIdArray,
-    useAvailabilityBlacklist: true,
+    blacklist: "availability",
```

Comment:
I am unsure whether this is as requested.

`useAvailabilityBlacklist: true` was mapped to the `both` type. Now it is the `availability` type.